### PR TITLE
Update Logs

### PR DESCRIPTION
**Pull request by nibanks**
Fixes a few things I noticed while looking at some logs. Also ran with dev mode so it resorted everything.

### CODE REVIEW

File: src/manifest/MsQuicEtw.man [hunk 1]
```diff
@@ -199,41 +199,49 @@
                 value="2"
                 />
             <map
-                message="$(string.Enum.QUIC_API_TYPE.STRM_CLOSE)"
+                message="$(string.Enum.QUIC_API_TYPE.SET_CONFIGURATION)"
```
Comment (thhous-msft) [inline src/manifest/MsQuicEtw.man:202]: This isn't going to break decoding old logs, right?

Comment (nibanks) [reply src/manifest/MsQuicEtw.man:202]: Kind of, but we don't have a choice because the enum in the code changed. I'd rather be correct with main.

Comment (thhous-msft) [reply src/manifest/MsQuicEtw.man:202]: Ok

**Review by thhous-msft — APPROVED** [review]

File: src/manifest/MsQuicEtw.man (hunk 1 above)
Comment (thhous-msft) [reply src/manifest/MsQuicEtw.man:202]: Then we should backport this to Ni.

Comment (nibanks) [reply src/manifest/MsQuicEtw.man:202]: Ok.

**Pull request merged**